src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetApprenticeship/GetApprenticeshipQueryHandler.cs

```diff
@@ -62,7 +62,9 @@ public async Task<GetApprenticeshipQueryResult> Handle(GetApprenticeshipQuery re
                             : default(long?),
                         OriginalStartDate = apprenticeship.OriginalStartDate,
                         ApprenticeshipEmployerTypeOnApproval = apprenticeship.Cohort.ApprenticeshipEmployerTypeOnApproval,
-                        MadeRedundant = apprenticeship.MadeRedundant,                        
+                        MadeRedundant = apprenticeship.MadeRedundant,
+                        EmailAddressConfirmedByApprentice = (apprenticeship.ApprenticeshipConfirmationStatus != null),
+                        EmailShouldBePresent = (apprenticeship.Cohort.EmployerAndProviderApprovedOn >= new DateTime(2021,9,10) && apprenticeship.ContinuationOfId == null),
```

Comment:
Sorry a silly question, should we also check for black list over here?

Reply:
As above